### ISSUE TITLE
Sidebar-Widget-playbox-widget error

### ISSUE THREAD

**Issue by JDesignEra** [Pe8er/Sidebar.Widget#16]
I am getting an error for the playbox widget.
```
[Error] Sidebar-widget-playbox-widget-index-coffee:
"Sidebar.widget/Playbox.widget/as/Get Current Track.applescript:2488:2493: script error: Expected end of line but found identifier. (-2741)
"
	redraw (client.js:6:15819)
	(anonymous function) (client.js:6:14620)
	(anonymous function) (client.js:6:26901)
	callback (client.js:6:9076)
	(anonymous function) (client.js:6:4581)
	emit (client.js:1:3389)
	onreadystatechange (client.js:6:10068)
[Error] Failed to load resource: the server responded with a status of 500 (Internal Server Error) (run, line 0)
```

**Comment by Pe8er** [comment]
I'm sorry but I'm developing this widget anymore. I suggest you check out the significantly (hopefully) improved variant of Playbox: https://github.com/Pe8er/Playbox.widget

**Comment by zorensen** [comment]
@Pe8er Possible to help to implement the new PlayBox into Sidebar.widget? Love the look of it and I want it to work. Tried myself, but I don't have the knowledge I'm afraid... :( Got it to work, somehow - but the album cover doesn't. All I did was some copy/paste from the new Playbox to the one included in Sidebar.widget.

Thanks for any help and great work - I would love to use Sidebar.widget and Playbox with it!
